mlflow/models/evaluation/base.py

```diff
@@ -775,7 +775,18 @@ def _start_run_or_reuse_active_run():
         yield active_run.info.run_id
 
 
-def _resolve_default_evaluator(model_type, evaluator_config):
+# NB: We often pass around evaluator name, config, and its instance together. Ideally, the
+# evaluator class should have name and config as class attributes, however, it was not
+# designed that way. Adding them while keeping backward compatibility is not trivial.
+# So, we use a dataclass to bundle them together.
+@dataclass
+class EvaluatorBundle:
+    name: str
+    evaluator: ModelEvaluator
+    config: Dict[str, Any]
+
+
+def _resolve_default_evaluator(model_type, default_config) -> List[EvaluatorBundle]:
```

Comment:
note: This function previously returns list of evaluator names, but updated to return list of `EvaluatorBundle` for convenience of the caller side.